06-plot-ggplot2.Rmd

```diff
@@ -356,22 +367,39 @@ of a fill legend would be set using `fill = "MyTitle"`.
 
 ```{r theme}
 ggplot(data = asia_oceania, mapping = aes(x = year, y = lifeExp, colour = continent)) +
-  geom_line() + facet_wrap( ~ country) +
+  geom_line() +
+  facet_wrap(facets = vars(country)) +
   labs(
     x = "Year",              # x axis title
     y = "Life expectancy",   # y axis title
     title = "Figure 1",      # main title of figure
     colour = "Continent"      # title of legend
   ) +
-  theme(axis.text.x = element_text(angle = 90, hjust = 1),
+  theme(axis.text.x = element_text(size = 5),
         plot.title = element_text(face = "bold"))
 ```
 
 > ## Tip
-> Note that we apply a "theme" definition to rotate
-> the x-axis labels to maintain readability.  Nearly everything in
+> Note that we apply a "theme" definition to customize 
+> the x-axis label sizes to maintain readability.  Nearly everything in
 > ggplot2 is customizable.
```

Comment:
Great addition showing the built-in themes. What do you think about a quick demo of `?theme` and `?ggtheme` to show the miriad of options and reinforce using `?` help?

Reply:
Yeah, I think that is a great idea. Do you think this needs to be captured in the bookdown or could it be an 'on the fly' element?